openedx/features/edly/context_processor.py

```diff
@@ -50,35 +48,38 @@ def edly_app_context(request):  # pylint: disable=unused-argument
     """
     Context processor responsible for edly.
     """
-    edly_app_context = cache.get(EDLY_CACHE_NAME)
-    if not edly_app_context:
-        edly_app_context = {}
-        edly_app_context.update(
+    edly_context = cache.get(EDLY_CACHE_NAME)
+    if not edly_context:
+        edly_context = {}
+        panel_services_notifications_url = configuration_helpers.get_value('PANEL_SERVICES_NOTIFICATIONS_URL', '')
```

Comment:
Rename this to `PANEL_SERVICES_NOTIFICATIONS_URL` to `PANEL_NOTIFICATIONS_BASE_URL` since its just the base domain for multi-tenant site's panel notifications

Reply:
Feedback Implemented